WcaOnRails/config/locales/en.yml

```diff
@@ -129,6 +129,9 @@ en:
         external_website: "Website"
         generate_website: "I would like to use WCA website instead of an external one for additional informations."
 
+        base_entry_fee_lowest_denomination: "Base entry fee in the lowest denomination"
```

Comment:
Just to avoid confusion for our delegates, could you actually expand this comment to explain that we don't have payment support yet, this is just one step towards it, and that the price will show up <wherever it shows up>. Maybe even link to https://github.com/thewca/worldcubeassociation.org/issues?q=is%3Aopen+is%3Aissue+label%3Apayments, so they don't come to us saying "well, what if my competition has a fee for each event?".